### PR TITLE
[API Compatibility] fix the loss docs

### PR DESCRIPTION
**Pull request by zty-king**
这些loss文档存在格式问题或示例不统一，此次全部统一

### CODE REVIEW

File: docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.nn.CosineEmbeddingLoss.md [hunk 1]
```diff
@@ -21,17 +21,77 @@ PyTorch 相比 Paddle 支持更多其他参数，具体如下：
...
+reduction 为'sum'
+
+```python
+# PyTorch 写法
+torch.nn.CosineEmbeddingLoss(weight=w, reduction='sum')
+
+# Paddle 写法
+paddle.nn.CosineEmbeddingLoss(weight=w, reduction='sum')
+```
+reduction = 'none'
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.nn.CosineEmbeddingLoss.md]: 这里是不是要忘删

File: docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.nn.HingeEmbeddingLoss.md [hunk 2]
```diff
@@ -27,17 +27,76 @@ PyTorch 相比 Paddle 支持更多其他参数，具体如下：
...
+reduction 为'sum'
+
+```python
+# PyTorch 写法
+torch.nn.HingeEmbeddingLoss(weight=w, reduction='sum')
+
+# Paddle 写法
+paddle.nn.HingeEmbeddingLoss(weight=w, reduction='sum')
+```
+reduction = 'none'
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.nn.HingeEmbeddingLoss.md]: 这里是不是要忘删

File: docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.nn.MultiLabelMarginLoss.md [hunk 3]
```diff
@@ -20,17 +20,77 @@ PyTorch 相比 Paddle 支持更多其他参数，具体如下：
...
+reduction 为'sum'
+
+```python
+# PyTorch 写法
+torch.nn.MultiLabelMarginLoss(weight=w, reduction='sum')
+
+# Paddle 写法
+paddle.nn.MultiLabelMarginLoss(weight=w, reduction='sum')
+```
+reduction = 'none'
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.nn.MultiLabelMarginLoss.md]: 这里是不是要忘删

**Review by zhwesky2010 — COMMENTED** [review]
再看一下还没有格式问题

File: docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.nn.CosineEmbeddingLoss.md (hunk 1 above)
Comment (zty-king) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.nn.CosineEmbeddingLoss.md]: Done

File: docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.nn.HingeEmbeddingLoss.md (hunk 2 above)
Comment (zty-king) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.nn.HingeEmbeddingLoss.md]: Done

File: docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.nn.MultiLabelMarginLoss.md (hunk 3 above)
Comment (zty-king) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.nn.MultiLabelMarginLoss.md]: Done

**Review by zhwesky2010 — APPROVED** [review]
LGTM
